manifests/npm.pp

```diff
@@ -8,7 +8,7 @@
   String $package           = $title,
   $source                   = 'registry',
   Array $uninstall_options  = [],
-  $home_dir                 = '/root',
+  $home_dir                 = undef,
```

Comment:
while we're at it, we could just as well give home_dir an `Optional[Stdlib::AbsPath]` type.